src/controllers/movies.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
+const fetch = require('node-fetch')
+
+
+const getMovies = (req, res) => {
+    res.sendFile(path.join(__dirname, '..', '..', 'public', 'movies.html'));
+};
+
+const getApiMovies = (req, res) => {
+    // fetch movies from other endpoint
+    // send it to front
+    console.log();
+    const url = `https://api.themoviedb.org/3/discover/movie?sort_by=popularity.desc&api_key=${process.env.API_KEY}`
+    fetch(url)
+        .then(data => data.json())
+        .then(data => res.json(data))
+        .catch(err => console.log(err))
```

Comment:
error handling `console.log` is not enough and btw it will keep the request pending since you are only logging the error without sending a response :upside_down_face: